src/Mutant.ts

```diff
@@ -88,27 +125,18 @@ export default class Mutant {
   insertMutatedFile = function(sourceFiles: string[]) {
     this.typeUtils.expectParameterArray(sourceFiles, 'Mutant', 'sourceFiles');
     var mutatedSrc = _.clone(sourceFiles);
-    var mutantSourceFileIndex = _.indexOf(mutatedSrc, this.getFilename());
-    mutatedSrc[mutantSourceFileIndex] = this.getMutatedFilename();
+    var mutantSourceFileIndex = _.indexOf(mutatedSrc, this.filename);
+    mutatedSrc[mutantSourceFileIndex] = this.mutatedFilename;
     return mutatedSrc;
   };
 
   /**
-   * Gets the name of the file which has been mutated.
-   * @function
-   * @returns {String} The name of the mutated file.
-   */
-  getFilename() {
-    return this.filename;
-  };
-
-  /**
    * Gets if the Mutant has the status KILLED.
    * @function
    * @returns {Boolean} True if the Mutant has the status KILLED.
    */
   hasStatusKilled() {
```

Comment:
I don't think we the methods `hasStatusKilled`, `setStatusKilled`, `hasStatusUntested`, `setStatusUntested`, `hasStatusSurvived`, `setStatusSurvived`, `setStatusTimedOut` and `hasStatusTimedOut` anymore, as we now have enumerations. This would save a lot code.